oracleAQ/java/Dockerfile

```diff
@@ -0,0 +1,6 @@
+mvn clean install
```

Comment:
for Java builds we've been building and then packaging in dockerfile. this prevents building every time the service is deployed and also allows us to change the JDK image to jre-slim. see inventory-helidon etc. for examples